software/earl/authenticator.go

```diff
@@ -490,10 +490,10 @@ func (a *FileBasedAuthenticator) userHasAccess(user *User, target Target) (AuthR
 	// open.
 	space_open_to_public := false
 
-	hour_from, hour_to := user.AccessHours()
-	current_hour := a.clock.Now().Hour()
+	time_from, time_to := user.AccessHours(time.Now())
```

Comment:
this should be clock.Now().

We already have the clock mocked out, which makes things predictable.